extern/boostd-data/couchbase/service.go

```diff
@@ -46,11 +46,11 @@ func (s *Store) Start(ctx context.Context) error {
 func (s *Store) AddDealForPiece(ctx context.Context, pieceCid cid.Cid, dealInfo model.DealInfo) error {
 	log.Debugw("handle.add-deal-for-piece", "piece-cid", pieceCid)
 
-	ctx, span := tracing.Tracer.Start(context.Background(), "store.add_deal_for_piece")
+	ctx, span := tracing.Tracer.Start(ctx, "store.add_deal_for_piece")
 	defer span.End()
 
 	defer func(now time.Time) {
-		log.Debugw("handled.add-deal-for-piece", "took", fmt.Sprintf("%s", time.Since(now)))
```

Comment:
I'm not sure we want to replace `fmt.Sprintf("%s", time.Since(now))` with time.Since(now)
If we don't do the sprintf it will always output the number of nanoseconds, whereas if it converts it to a string it will output a humanized string like "100ms"

Reply:
Ah, good to know, I'll take a look at this. These were all changes to appease the linter, will find a better way to resolve this.

Reply:
I think we can probably just do time.Since(now).String()

Reply:
It does indeed, sample output - `{"took": "2.840023375s"},{"took": "365.096458ms"}` _(not actual operation times)_. The previous one displayed a float of seconds regardless of duration.